app/src/main/java/net/gsantner/markor/format/markdown/MarkdownAutoFormat.java

```diff
@@ -67,7 +67,7 @@ private String createIndentForNextLine(Spanned dest, int dend, int istart) {
                 // This is for any line that is not the first line in a file
                 Matcher listMatcher = MarkdownHighlighterPattern.LIST_UNORDERED.pattern.matcher(dest.toString().substring(iend, dend));
                 if (listMatcher.find()) {
-                    return dest.subSequence(istart, iend) + Character.toString(dest.charAt(iend)) + " ";
```

Comment:
have you checked that all textactions using this stilll do work?

Reply:
I have checked that auto completion works for unordered list (-, *) and ordered list (1., ...) as well as for checkboxes (- [ ], - [x]). As I don't know what the function covers else I am not sure if that's all. Hoped tp get a hint/finding from your side during the review :-) Is that all what the function covers or is there something missing?